### PR TITLE
php8 compatibility/fix integration test Magento_Swatches module

### PR DESCRIPTION
**Pull request by SilinMykola**
### Description (*)
Fix integration test `Magento/Swatches/Controller/Adminhtml/Product/AttributeTest.php `

### Related Issues
1. magento/magento2#34020

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by andrewbess** [comment]
@magento run all tests

File: app/code/Magento/Swatches/Model/Plugin/EavAttribute.php [hunk 1]
```diff
@@ -350,7 +358,13 @@ protected function processTextualSwatch(Attribute $attribute)
                     //option was deleted by button with basket
                     continue;
                 }
+                if (!is_array($storeValues)) {
+                    throw new Exception(
+                        __('Invalid attribute type')
+                    );
```
Comment (Den4ik) [inline app/code/Magento/Swatches/Model/Plugin/EavAttribute.php]: I believe it should be `LocalizedException`

**Review by Den4ik — CHANGES_REQUESTED** [review]
Please look at review comment

**Comment by andrewbess** [comment]
@magento run all tests

**Comment by andrewbess** [comment]
@magento run all tests with env PHP 8.0

**Pull request merged**